content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -0,0 +1,233 @@
+    <title>Wikipedia, the free encyclopedia</title>
+    {{< /text >}}
+
+1.  Check the access log of the proxy and see your request:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -n external -l app=squid -o jsonpath={.items..metadata.name}) -n external -- tail -f /var/log/squid/access.log
+    1544160065.248    228 172.30.109.89 TCP_TUNNEL/200 87633 CONNECT en.wikipedia.org:443 - HIER_DIRECT/91.198.174.192 -
+    {{< /text >}}
+
+## Configure traffic to external HTTPS proxy
+
+1.  Define a TCP (!) Service Entry for the HTTPS proxy. Note that despite the fact that HTTP
+    [CONNECT](https://tools.ietf.org/html/rfc7231#section-4.3.6) method is used to communicate with HTTPS proxies, the traffic between the application and the proxies is TCP (a TCP tunnel), and not HTTP.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: ServiceEntry
+    metadata:
+      name: proxy
+    spec:
+      hosts:
+      - my-company-proxy.com # ignored
+      addresses:
+      - $PROXY_IP/32
+      ports:
+      - number: $PROXY_PORT
+        name: tcp
+        protocol: TCP
+      location: MESH_EXTERNAL
+    EOF
+    {{< /text >}}
+
+1.  Send a request from the `sleep` pod in the `default` namespace. The `sleep` pod has Istio sidecar injected and its
+    traffic is controlled by Istio.
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- sh -c "HTTPS_PROXY=$PROXY_IP:$PROXY_PORT curl https://en.wikipedia.org/wiki/Main_Page" | grep -o "<title>.*</title>"
+    <title>Wikipedia, the free encyclopedia</title>
+    {{< /text >}}
+
+1.  Check the Istio sidecar proxy's logs and see your request appear:
+
+    {{< text bash >}}
+    $ kubectl logs $SOURCE_POD -c istio-proxy
+    [2018-12-07T10:38:02.841Z] "- - -" 0 - 702 87599 92 - "-" "-" "-" "-" "172.30.109.95:3128" outbound|3128||my-company-proxy.com 172.30.230.52:44478 172.30.109.95:3128 172.30.230.52:44476 -
+    {{< /text >}}
+
+1.  Check the access log of the proxy and see your request:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -n external -l app=squid -o jsonpath={.items..metadata.name}) -n external -- tail -f /var/log/squid/access.log
+    1544160065.248    228 172.30.109.89 TCP_TUNNEL/200 87633 CONNECT en.wikipedia.org:443 - HIER_DIRECT/91.198.174.192 -
+    {{< /text >}}
+
+## Understanding what happened
+
+In this example you configured access to an external legacy HTTPS proxy from the Istio service mesh. To simulate a
+remote HTTPS proxy, you deployed an HTTPS proxy in a separate namespace, without Istio automatic sidecar injection. In addition, for simulation of a remote proxy, you addressed the HTTPS proxy by its IP address and not by a Kubernetes
+service.
+
+To enable Istio-controlled traffic to the external HTTPS proxy you created a TCP service entry with the IP address and
+ the port of the proxy.
+
```

Comment:
I think you should elaborate here. For example, explain what is fundamentally different than when you don't use a proxy like why didn't you need to create service entries for the external host (en.wikipedia.org or wildcard), like you would normally do.